doc/qbk/6.2.MutableString.qbk

```diff
@@ -0,0 +1,70 @@
+[/
+    Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:mutablestring MutableString]
+
+A ['MutableString] is a container that has the `assign` and
+`append` member functions that accept iterators to `char`.
+
+The class __pct_encoded_view__ provides a operations
+to assign and append percent-decoded values to a
+['MutableString].
```

Comment:
Grammar

```
Objects of type __pct_encoded_view__ reference URL or other component strings that are percent-encoded. That is, that characters which are special (not in the allowed character set) are represented with escapes that start with a percent followed by a two-digit hexadecimal number of the corresponding character code (which may be part of a UTF-8 code point depending on the context). The primary operations are:

* Comparison to other percent-encoded or plain strings
* Appending the decoded characters to an existing container
* Assigning the decoded characters to an existing container
* Iterating over the represented decoded character range
* Accessing the underlying percent-encoded string
...
```

Reply:
"These objects can only be constructed from strings that have a valid percent-encoding, otherwise construction fails."

This should probably go on the main class javadoc